grails-app/controllers/org/pih/warehouse/api/StockMovementItemApiController.groovy

```diff
@@ -55,7 +52,7 @@ class StockMovementItemApiController {
         log.debug "JSON " + jsonObject.toString(4)
         StockMovementItem stockMovementItem = stockMovementService.getStockMovementItem(params.id)
 
-        stockMovementService.removeShipmentItemsForModifiedRequisitionItem(stockMovementItem)
+        stockMovementService.removeShipmentAndPicklistItemsForModifiedRequisitionItem(stockMovementItem)
```

Comment:
This is fine for now, but I'd love to come up with a less verbose naming convention for some of these methods. Is there a way to describe this in fewer words? 
```
refreshOnUpdate() 
resetOnUpdate() 
```
or maybe even 
```
updatePicklist()
```
since ultimately that's what we're doing.


I had the same problem when naming the methods in InventoryService and now I want to go back to a simple name to describe what we're doing. For example, getQuantity() or getQuantityDetails() rather than awful names like getProductQuantityByInventoryItemMap().